Framework/Kernel/inc/MantidKernel/PropertyManagerProperty.h

```diff
@@ -0,0 +1,58 @@
+#ifndef MANTID_KERNEL_PROPERTYMANAGERPROPERTY_H_
+#define MANTID_KERNEL_PROPERTYMANAGERPROPERTY_H_
+
+#include "MantidKernel/PropertyWithValue.h"
+#include "MantidKernel/PropertyManager_fwd.h"
+
+namespace Mantid {
+namespace Kernel {
+
+/**
+
```

Comment:
Later on it would be nice to have here a few words about what this class is meant for.